lambdas/jwtauthorizationrequest/src/test/java/uk/gov/di/ipv/cri/passport/jwtauthorizationrequest/JwtAuthorizationRequestHandlerTest.java

```diff
@@ -228,6 +229,27 @@ void shouldReturn302WhenValidationFails() throws Exception {
                 errorResponse.getDescription());
     }
 
+    @Test
+    void shouldReturn302WithRedirectUriWhenValidationFailsButIsRecoverable() throws Exception {
+        when(jarValidator.validateRequestJwt(any(), anyString()))
+                .thenThrow(
+                        new RecoverableJarValidationException(
+                                OAuth2Error.INVALID_REQUEST_OBJECT, "http://redirect-url.com"));
+
+        var event = new APIGatewayProxyRequestEvent();
+        Map<String, String> map = new HashMap<>();
+        map.put("client_id", "TEST");
+        event.setHeaders(map);
+        event.setBody(signedJWT.serialize());
+
+        var response = underTest.handleRequest(event, context);
+
+        assertEquals(302, response.getStatusCode());
+        assertEquals(
+                "\"{\\\"redirect_uri\\\":\\\"http://redirect-url.com\\\",\\\"oauth_error\\\":{\\\"error_description\\\":\\\"Invalid request JWT\\\",\\\"error\\\":\\\"invalid_request_object\\\"}}\"",
```

Comment:
The triple backslash seems a bit off here? Not sure if this is like double excaping the characters I think? This could be to do with my previous comment since objectMapper.writeValueAsString is being called twice at the moment

Reply:
You're right yeah, fixed now